src/helpers/htmldomapi.ts

```diff
@@ -0,0 +1,61 @@
+import {DOMAPI} from '../types';
```

Comment:
@staltz maybe you can provide some feedback here. The git diff is not that nice since it didn't detect the move operation, but in short, the changes in this file are around the fact that `toVNode()` only operates in DOM structures, and thats why I have removed few of the old APIs that were only used by `toVNode()` from this file.

Do you foresee `helpers/tovnode` operating on non-DOM trees? IMO if you are using a non-DOM tree, you can implement your own tovnode routine to build such tree.

Reply:
That's a good point, when I wrote `toVNode` I introduced the `domApi` argument just because `toVNode` happened to do similar kinds of lookups that DOMAPI objects had, but I didn't argue for some use case where the domApi is something else than the htmldomapi.

So I'm okay with these changes.

On a bigger picture perspective, `DOMAPI` serves as an abstraction on how to **write** the DOM. `toVNode` needed an abstraction on how to **read** the DOM. In the typical use case for snabbdom, reading the DOM tree is not necessary since the VDOM is used for reading instead. So the VDOM is the source of truth. We could ask ourselves overall what are those cases where we need to read specifically from the DOM and not from the VDOM. If there are enough use cases, then it could justify an abstraction layer for reading the DOM, and perhaps reintroducing that back into `toVNode`.

Reply:
excellent! thanks.